app/uk/gov/hmrc/economiccrimelevyregistration/controllers/AmendReasonController.scala

```diff
@@ -46,8 +46,7 @@ class AmendReasonController @Inject() (
     with I18nSupport
     with BaseController
     with ErrorHandler {
-  val form: Form[String] = formProvider()
-
+  val form: Form[String]                         = formProvider()
```

Comment:
I don't think we need to revert this change as previously only space was removed